bundles/http/Citrix/Microsoft-Outlook/script.js

```diff
@@ -1,230 +1,1198 @@
+    dataStore,
+    client,
+    actionParameters,
+}) {
+    const startDate = moment.utc().subtract(30, 'm').format();
+    const responseOfcreateOutlookOneTimeEventCustomTimezone = await client.fetch(
+        `/v1.0/me/events`,
+        {
+            method: "POST",
+            body: JSON.stringify({
+
+                subject: actionParameters.subject,
+                body: {
+                    contentType: "HTML",
+                    content: actionParameters.content
+                },
+                start: {
+                    dateTime: `${actionParameters.startDate}T${actionParameters.startTime}`,
+                    timeZone: actionParameters.timezone
+                },
+                end: {
+                    dateTime: `${actionParameters.endDate}T${actionParameters.endTime}`,
+                    timeZone: actionParameters.timezone
+                },
+                location: {
+                    displayName: actionParameters.location
+                },
+                attendees: [
+                    {
+                        emailAddress: {
+                            address: actionParameters.email1
+                        },
+                        type: actionParameters.type1
+                    },
+                    {
+                        emailAddress: {
+                            address: actionParameters.email2
+                        },
+                        type: actionParameters.type2
+                    }, {
+                        emailAddress: {
+                            address: actionParameters.email3
+                        },
+                        type: actionParameters.type3
+                    }, {
+                        emailAddress: {
+                            address: actionParameters.email4
+                        },
+                        type: actionParameters.type4
+                    },
+                    {
+                        emailAddress: {
+                            address: actionParameters.email5
+                        },
+                        type: actionParameters.type5
+                    }, {
+                        emailAddress: {
+                            address: actionParameters.email6
+                        },
+                        type: actionParameters.type6
+                    }
+                ],
+                allowNewTimeProposals: actionParameters.allowNewTimeProposals,
+                isOnlineMeeting: actionParameters.isOnlineMeeting,
+                onlineMeetingProvider: actionParameters.onlineMeetingProvider
+
+            }),
+        }
+    );
+    //console.log(JSON.stringify(responseOfcreateOutlookOneTimeEventCustomTimezone));
+    if (!responseOfcreateOutlookOneTimeEventCustomTimezone.ok) {
+        throw new Error(`createOutlookOneTimeEventCustomTimezone (${responseOfcreateOutlookOneTimeEventCustomTimezone.status}: ${responseOfcreateOutlookOneTimeEventCustomTimezone.statusText})`);
```

Comment:
Update the error message as per the previous comment. 

> Error logs should give enough information to the end user.
Ex: `${ServiceActionName} Service action is failed with the error ${status}:${statusText}`